gradle/dependency-updates.gradle

```diff
@@ -6,25 +6,16 @@ def groovyVersion = project.rootProject
         .get()
         .displayName
 
-def micronautVersion = project.rootProject
+
+def grailsVersion = project.rootProject
         .extensions
         .getByType(VersionCatalogsExtension.class)
         .named("libs")
-        .findVersion("micronaut")
+        .findVersion("grails")
         .get()
         .displayName
 
-allprojects {
-    configurations.configureEach {
-        resolutionStrategy.eachDependency { DependencyResolveDetails details ->
-            if ((details.requested.group == 'org.codehaus.groovy' || details.requested.group == 'org.apache.groovy') && details.requested.name != 'groovy-bom') {
-                details.useTarget(group: 'org.apache.groovy', name: details.requested.name, version: groovyVersion)
-                details.because "The dependency coordinates are changed in Apache Groovy 4, plus ensure version"
-            }
 
-            if (details.requested.group == "io.micronaut" && details.requested.name == "micronaut-inject-groovy") {
-                details.useVersion(micronautVersion)
-            }
-        }
-    }
+dependencies {
+    implementation platform("org.grails:grails-bom:$grailsVersion")
```

Comment:
Why is this needed? Should we not add the bom to the TOML-file instead?
See https://github.com/gpc/grails-mail/blob/5.0.x/gradle/libs.versions.toml for reference.

Reply:
Just an oversight on my part, will put this in TOML for now.

Reply:
That was the first change I made before remembering this was on TOML.